crypto/rand/sysrand.c

```diff
@@ -58,11 +58,15 @@ int GFp_sysrand_chunk(void *buf, size_t len);
 
 #pragma warning(pop)
 
-const size_t GFp_sysrand_chunk_max_len = ULONG_MAX;
-
 int GFp_sysrand_chunk(void *out, size_t requested) {
```

Comment:
See below. the return type needs to be `long` and we need to use `LONG_MAX` and `long` everywhere here.